library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -81,6 +83,17 @@ data class Conversations(
         )
     }
 
+    suspend fun newGroup(accountAddress: String): Group {
+        val group = libXMTPConversations?.createGroup(accountAddress)
```

Comment:
I think it's a mistake in the bindings that `createGroup` takes a single address.

We could either do no address or take an array of addresses (and pre-populate the group for the caller).

Reply:
I think for usage it would probably make more sense for it to take an array of addresses